backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Controllers/ResourceController.cs

```diff
@@ -43,19 +43,17 @@ public class ResourceController : ControllerBase
         /// <summary>
         /// Get list of maskinprotenschema resources
         /// </summary>
-        /// <param name="party">the partyid</param>
-        /// <returns></returns>
+        /// <returns>List of API service resources</returns>
         [HttpGet]
-        [Route("accessmanagement/api/v1/{party}/resources/maskinportenschema")]
-        public async Task<ActionResult<List<ServiceResourceFE>>> Get([FromRoute] int party)
+        [Authorize]
+        [Route("accessmanagement/api/v1/resources/maskinportenschema")]
+        public async Task<ActionResult<List<ServiceResourceFE>>> Get()
         {
-            string languageCode = "nb";
+            int userId = AuthenticationHelper.GetUserId(_httpContextAccessor.HttpContext);
+            UserProfile userProfile = await _profileService.GetUserProfile(userId);
+            string languageCode = userProfile?.ProfileSettingPreference?.Language ?? "nb";
```

Comment:
vi har en hjelper metode som gjør det string languageCode = ProfileHelper.GetLanguageCodeForUser(userProfile);

Reply:
Yes, but that only returns correct language code based on the name of the language files for the application (no_nb, no_nn, en) while resource-registry (and other A3 implementations) use the 2-character language codes as returned by the SBL Bridge user profile integration.

Reply:
Ahh, sorry my bad. I thought you referred to the GetStandardLanguageCodeForUser.
I can update to use the ProfileHelper.GetLanguageCodeForUser(userProfile);

Reply:
yeah :)